src/lib.rs

```diff
@@ -268,6 +276,26 @@ impl ProgLog {
         }
     }
 
+    /// Increment the progress logger by `count`, checking if new message(s) should be emitted.
+    /// This is equivalent to calling the `record` function `count` number of times.
```

Comment:
```suggestion
    /// Emits `count // unit` messages, there is no carry over of remainder counts between calls.
```

suggestion: I really think it's important to note that calling `record` 10 times with unit of 2 will emit 5 messages, but calling this with `count=5` twice will emit 4 messages.